stock_buffer_capacity_limit/models/stock_buffer.py

```diff
@@ -16,7 +16,12 @@ class StockBuffer(models.Model):
         "execution alerts.",
     )
 
-    @api.depends("storage_capacity_limit")
+    @api.depends(
+        "storage_capacity_limit",
+        "product_location_qty",
```

Comment:
`product_location_qty` AFAIS is not stored, you cannot use it here

Reply:
You are sure? I thought we could, at least to trigger the computation from the UI like would do an onchange. Maybe I'm wrong.

Reply:
normally I get a warning from server logs complaining the the field is not stored and, as such, is useless to hook to its change. Am I missing anything?

Reply:
It is not stored indeed.
But I did not know that rule either.
Removed from the depends...

Reply:
bah I cannot find any reference in the code...but https://github.com/odoo/odoo/issues/37468
It seems it happens only if the field has no depends specified?
`_compute_product_available_qty` has no depends.... who wins? :smile:

Reply:
you do